Doc/library/importlib.metadata.rst

```diff
@@ -113,7 +113,7 @@ Entry points
 
 The ``entry_points()`` function returns a collection of entry points.
 Entry points are represented by ``EntryPoint`` instances;
-each ``EntryPoint`` has a ``.name``, ``.group``, and ``.value`` attributes and
+each ``EntryPoint`` has ``.name``, ``.dist``, ``.group``, and ``.value`` attributes and
```

Comment:
I'm not sure I'd want to document `EntryPoint` this way. Some `EntryPoint` objects have a `.dist` but others do not (see the docstring for `Entrypoint.matches`). Notice also that `EntryPoint._key()` (and methods relying on it) do not consider `.dist` to be essential.